snp_motion_planning/launch/planning_server.launch.xml

```diff
@@ -19,7 +20,7 @@
   <arg name="max_translational_acc" default="0.100"/>
   <arg name="max_rotational_acc" default="3.14"/>
   <arg name="check_joint_accelerations" default="false"/>
-  <arg name="contact_check_distance" default="0.0"/>
+  <arg name="min_contact_distance" default="0.0"/>
```

Comment:
I know this is what we had before, but a default contact distance of 0.0 seems too close to me.